### ISSUE TITLE
ParamAttr的疑问

### ISSUE THREAD

**Issue by starsblinking** [PaddlePaddle/Paddle#28010]
在 https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/fluid_cn/ParamAttr_cn.html#paramattr 的描述中，有:
1. initializer (Initializer，可选) - 参数的初始化方式。默认值为None，表示权重参数采用Xavier初始化方式，偏置参数采用全0初始化方式。
请问这里的"权重参数"和“偏置参数”是什么意思啊？感觉像是在有y=wx+b这样类似的layer中才有的概念？

2. learning_rate (float) - 参数的学习率。实际参数的学习率等于全局学习率乘以参数的学习率，再乘以learning rate schedule的系数。
假如我设置这里的“参数的学习率"为learning_rate_param_attr, 使用学习率为learning_rate_adam的Adam优化器, 那么参数在更新的时候依照的还是 https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/optimizer_cn/AdamOptimizer_cn.html#cn-api-fluid-optimizer-adamoptimizer 中的公式，只不过其中的learning_rate由learning_rate_adam变成了learning_rate_adam*learning_rate_param_attr，对吗?
另外，这里的 learning rate schedule 是什么意思啊？

**Comment by jiweibo** [comment]
> 在 https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/fluid_cn/ParamAttr_cn.html#paramattr 的描述中，有:
> 
> 1. initializer (Initializer，可选) - 参数的初始化方式。默认值为None，表示权重参数采用Xavier初始化方式，偏置参数采用全0初始化方式。
>    请问这里的"权重参数"和“偏置参数”是什么意思啊？感觉像是在有y=wx+b这样类似的layer中才有的概念？
> 2. learning_rate (float) - 参数的学习率。实际参数的学习率等于全局学习率乘以参数的学习率，再乘以learning rate schedule的系数。
>    假如我设置这里的“参数的学习率"为learning_rate_param_attr, 使用学习率为learning_rate_adam的Adam优化器, 那么参数在更新的时候依照的还是 https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/optimizer_cn/AdamOptimizer_cn.html#cn-api-fluid-optimizer-adamoptimizer 中的公式，只不过其中的learning_rate由learning_rate_adam变成了learning_rate_adam*learning_rate_param_attr，对吗?
>    另外，这里的 learning rate schedule 是什么意思啊？

1. 权重参数和偏置参数就是你理解的weights和bias的参数

2. 你理解的是正确的，每个参数的学习率是优化器设置的lr 乘以 ParamAttr中设置的lr。 learning rate schedule我理解的就是和优化算法相关的概念了，比如adam中动态的调整lr
